src/Eshopworld.Messaging/Eshopworld.Messaging.csproj

```diff
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
     
-    <Version>2.1.0</Version>
-    <PackageReleaseNotes>Added ability to override event names using EswEventName attribute</PackageReleaseNotes>
+    <Version>2.1.1-preview</Version>
```

Comment:
@VaclavK now that is has been tested, do you plan on going with the non-preview version?

Reply:
yeah the approach was to remove the preview - the preview was only to facilitate the peter pan perf test - resolved now